### DISCUSSION TITLE
How to choose wisely for env in meta config

### DISCUSSION THREAD

**Discussion by nhattran998** [vikejs/vike#847] [Discussion]
https://vite-plugin-ssr.com/meta

Currently, I just know when config client/server --> Value only/both available on the client/server side. But I do not know when to choose server, client, or both. Can you add some docs to explain this?

Thanks 🙏

**Comment by brillout** [comment]
I'm not sure there can be a generic guide about this: it really depends on what kind of custom config you're implementing. Note that `meta` will eventually be used only by integrators (`vike-react`, `vike-solid`, ...) and end-users won't have to use `meta` at all.

**Reply by nhattran998** [reply]
Hmm - can you have some guidelines or documents for this? I know this is not your high priority right now, but at least - have some guide here can lead to end-users understanding 🤔.

**Reply by brillout** [reply]
What is it you're confused about?

**Reply by brillout** [reply]
(Thanks for sponsoring 💚. Next step is to convince your employer to sponsor 🙂.)
